eth/main.cpp

```diff
@@ -340,6 +341,10 @@ int main(int argc, char** argv)
 	/// Wallet password stuff
 	string masterPassword;
 
+	// make sure data dir exists
+	if (!boost::filesystem::create_directory(getDataDir()))
+		BOOST_THROW_EXCEPTION(FileError());
```

Comment:
Does create_directory throw?
As this is CLI, would rather see an error message output to the user with exit() rather than a crash.